doc.go

```diff
@@ -1,3 +1,6 @@
+// Copyright Some Company Corp
```

Comment:
Thank you!

Can we make it more explanatory? Kind of like what you posted in the issue. I'd generalize and call that a file header maybe. And say something like "This is a file header. It is separated from the package doc with an empty line so it is ignored by Godoc. Can be used for copyright or internal description of the file."

Does that sound good?